open-sphere-base/core/src/main/java/io/opensphere/core/cache/CacheOptionsProvider.java

```diff
@@ -121,14 +121,12 @@ public JPanel getOptionsPanel()
 
         mySizeLimitEnabledCheckbox = new JCheckBox();
         mySizeLimitEnabledCheckbox.setSelected(sizeLimitBytes > 0L);
-        mySizeLimitEnabledCheckbox.addChangeListener(new ChangeListener()
+        mySizeLimitEnabledCheckbox.addActionListener(e ->

```

Comment:
Does converting this from a change listener to an action listener affect it's behavior?

Reply:
There are some times when it might do different things, so I left the change listener with the previous behavior and added an action listener for the metrics collection.